src/cli.spec.ts

```diff
@@ -1,342 +1,83 @@
-import { removeFromGlobalConfig } from './utils/config'
 import { cli } from './cli'
-import {
-  createTestApp,
-  createTestGlobalTarget,
-  mockProcessExit,
-  removeTestApp
-} from './utils/test'
-import * as mainModule from './main'
-import { ReturnCode } from './main'
-import { Command } from './utils/command'
-import { Target, generateTimestamp } from '@sasjs/utils'
+import { mockProcessExit } from './utils/test'
+import * as envVariablesModule from './utils/loadEnvVariables'
+import * as projectDirModule from './utils/setProjectDir'
+import * as parseModule from './types/command/parse'
+import { Logger } from '@sasjs/utils'
+import { CommandBase } from './types/command/commandBase'
+import { ReturnCode } from './types/command'
+
+const args = ['/usr/local/bin/node', '/usr/local/bin/sasjs', 'mock', 'command']
+
+class MockCommand extends CommandBase {
+  constructor(args: string[]) {
+    super(args, {
+      syntax: 'mock <command>'
+    })
+  }
+  public async execute() {
+    return 0
+  }
+}
+
+const mockCommand = new MockCommand(args)
 
 describe('CLI command parsing', () => {
```

Comment:
We need not test command parsing at this level since that is handled by the individual command classes.